chunkflow/operators/arbitrary_op.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from chunkflow.chunk import Chunk
+from .operator_base import OperatorBase
+
+from .block_inference.frameworks.pytorch_patch_inference_engine import load_source
+
+
+class ArbitraryChunkOperator(OperatorBase):
+    r"""
+    """
+    def __init__(self,
+                 opprogram: str=None,
+                 args: str=None,
+                 name: str='arbitrary-chunk-op'):
```

Comment:
I normally have verbose parameter here. I see you are using `debug` parameter later. To be consistent, you can add a boolean flag `verbose` here.